clouddriver-integration/src/test/java/com/netflix/spinnaker/clouddriver/BaseContainerTest.java

```diff
@@ -5,110 +5,64 @@
+import org.junit.jupiter.api.*;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.testcontainers.containers.GenericContainer;
 import org.testcontainers.containers.Network;
-import org.testcontainers.containers.output.Slf4jLogConsumer;
 import org.testcontainers.containers.wait.strategy.Wait;
 import org.testcontainers.junit.jupiter.Testcontainers;
 import org.testcontainers.utility.DockerImageName;
 
 @Testcontainers
-class StandaloneContainerTest {
-
-  private static final String REDIS_NETWORK_ALIAS = "redisHost";
-
-  private static final int REDIS_PORT = 6379;
+public class BaseContainerTest {
 
-  private static final Logger logger = LoggerFactory.getLogger(StandaloneContainerTest.class);
+  private static final Logger logger = LoggerFactory.getLogger(BaseContainerTest.class);
 
-  private static final Network network = Network.newNetwork();
+  protected static final Network network = Network.newNetwork();
 
-  private static final GenericContainer redis =
-      new GenericContainer(DockerImageName.parse("library/redis:5-alpine"))
-          .withNetwork(network)
-          .withNetworkAliases(REDIS_NETWORK_ALIAS)
-          .withExposedPorts(REDIS_PORT);
-
-  private static GenericContainer clouddriverContainer;
+  protected static GenericContainer clouddriverContainer;
 
   @BeforeAll
-  static void setupOnce() throws Exception {
+  static void setupInit() throws Exception {
     String fullDockerImageName = System.getenv("FULL_DOCKER_IMAGE_NAME");
 
     // Skip the tests if there's no docker image.  This allows gradlew build to work.
     assumeTrue(fullDockerImageName != null);
 
-    redis.start();
-
     DockerImageName dockerImageName = DockerImageName.parse(fullDockerImageName);
 
     clouddriverContainer =
         new GenericContainer(dockerImageName)
             .withNetwork(network)
             .withExposedPorts(7002)
-            .dependsOn(redis)
-            .waitingFor(Wait.forHealthcheck().withStartupTimeout(Duration.ofSeconds(90)))
-            .withEnv("SPRING_APPLICATION_JSON", getSpringApplicationJson());
-
-    Slf4jLogConsumer logConsumer = new Slf4jLogConsumer(logger);
-    clouddriverContainer.start();
-    clouddriverContainer.followOutput(logConsumer);
-  }
-
-  private static String getSpringApplicationJson() throws JsonProcessingException {
-    String redisUrl = "redis://" + REDIS_NETWORK_ALIAS + ":" + REDIS_PORT;
-    logger.info("redisUrl: '{}'", redisUrl);
-    Map<String, String> properties =
-        Map.of("redis.connection", redisUrl, "services.fiat.baseUrl", "http://nowhere");
-    ObjectMapper mapper = new ObjectMapper();
-    return mapper.writeValueAsString(properties);
-  }
-
-  @AfterAll
-  static void cleanupOnce() {
-    if (clouddriverContainer != null) {
-      clouddriverContainer.stop();
-    }
-
-    if (redis != null) {
-      redis.stop();
-    }
+            .waitingFor(Wait.forHealthcheck().withStartupTimeout(Duration.ofSeconds(90)));
```

Comment:
This fails for me locally with a 90 second timeout, but passes with 120.  Mind changing it to:
```suggestion
            .waitingFor(Wait.forHealthcheck().withStartupTimeout(Duration.ofSeconds(120)));
```
I suppose at some point different tests could use different timeouts, but maybe we're not quite at that point yet.

Reply:
Done.